libs/design/paginator/examples/src/public_api.ts

```diff
@@ -1,5 +1,4 @@
 export { BasicPaginatorComponent } from './basic-paginator/basic-paginator.component';
 export { LinkPaginatorComponent } from './link-paginator/link-paginator.component';
 
-export { PaginatorExamplesModule } from './paginator-examples.module';
-export { PAGINATOR_EXAMPLES } from './examples';
+export * from './examples';
```

Comment:
this file isn't set up like all the others and is causing errors

Reply:
hm odd, that error doesn't throw on my machine, even when rimrafing `dist`. Can you try again with the pushed commit?

Reply:
paginator was missing an `index.ts`. I added it and `daffio-dev` served correctly